Earthfile

```diff
@@ -255,6 +255,10 @@ chaos-test:
     FROM +code
     RUN go test -tags chaos ./...
 
+offline-test:
+    FROM +code
+    RUN --network=none go test -tags offline ./...
```

Comment:
This could alternatively be `RUN --network=none go test ./cloud -run TestClientConnectionIsLazy`. Or we could use an onpar suite to allow multiple tests with the `-run` flag, e.g.

```go
func TestOffline(t *testing.T)  {
    o := onpar.New(t)
    defer  o.Run()
    o.Spec("ClientConnectionIsLazy", func(t *testing.T) {
         // ...
    })
}
```

Then `go test -run TestOffline` would run all of the offline subtests.

Reply:
Thanks for the idea -- I like it more than using a tag.